trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -468,8 +482,10 @@ def make_spectrum(self, input_object, output_file=None,
         input_ds.domain_left_edge = input_ds.domain_left_edge.to('code_length')
         input_ds.domain_right_edge = input_ds.domain_right_edge.to('code_length')
 
-        if self.bin_space == 'velocity':
```

Comment:
So we won't be able to do in velocity space?  Seems like leaving this in would allow for it, but maybe not?